### PR TITLE
[StemSeries] Add example

### PR DESCRIPTION
**Pull request by seijikun**

### CODE REVIEW

File: ExampleGenerator/Series/StemSeriesExamples.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+	public class StemSeriesExamples
+	{
+		[Export(@"Series\StemSeries")]
+		public static PlotModel StemSeries()
+		{
+
+			var model = new PlotModel{ Title = "Fun with Bats" };
+
+			Func<double, double>[] batFn = new Func<double, double>[]
```
Comment (objorke) [inline ExampleGenerator/Series/StemSeriesExamples.cs]: I think the functions are complicating the example here. Can you find some other discrete sequence data to demonstrate stem series? http://se.mathworks.com/help/matlab/ref/stem.html

**Comment by seijikun** [comment]
Would it be ok to add a simpler one, but leave this one in?

File: ExampleGenerator/Series/StemSeriesExamples.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+			model.Series.Add(sinStemSeries);
+			model.Series.Add(cosStemSeries);
+
+			return model;
+		}
+
+
+		[Export(@"Series\StemSeries_advanced")]
+		public static PlotModel StemSeries_advanced()
```
Comment (objorke) [inline ExampleGenerator/Series/StemSeriesExamples.cs]: No new functionality demonstrated, so I don't think this is a good example. Even if it is a fun function :-)
I think focus should be on the StemSeries functionality (e.g. the `Base` property), and keep the other code to a minimum.

Comment (objorke) [inline ExampleGenerator/Series/StemSeriesExamples.cs]: Showing different data providers (add to `Points`, set `ItemsSource`) could be a good example

File: ExampleGenerator/Series/StemSeriesExamples.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	using System;
+	using OxyPlot;
+	using OxyPlot.Axes;
+	using OxyPlot.Series;
+
+	public class StemSeriesExamples
+	{
+
+		[Export(@"Series\StemSeries")]
+		public static PlotModel StemSeries()
```
Comment (objorke) [inline ExampleGenerator/Series/StemSeriesExamples.cs]: I think the basic example needs just a single series. Could also use default values for the marker style, I think.

**Comment by seijikun** [comment]
Closing it here. Will make a new Pull Request

**Pull request closed without merging**